iconservice/iiss/reward_calc/storage.py

```diff
@@ -51,6 +67,10 @@ def open(self, path: str):
         self._db = KeyValueDatabase.from_path(current_db_path, create_if_missing=True)
         self._db_iiss_tx_index = self._load_last_transaction_index()
 
+        self.current_version, self.current_revision = self._load_version_and_revision()
```

Comment:
I think it should not be member veriable because it already asigned at self._load_version_and_revision()